<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductionPlanning</class>
 <widget class="QWidget" name="ProductionPlanning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>1027</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>691</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Segoe Script&quot;;
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>Les fermiers présentent à Félix les différents types 
 de produits qu'ils produisent à la coopérative,
 ainsi que le bénéfice qu'ils tirent de la vente de 
 chaque produit et le temps nécessaire pour 
 produire chaque unité du produit.</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>1071</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Segoe Script&quot;;
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>la coopérative pourrait avoir une capacité 
 maximal Resource pour la production                        Resource(temps ,hect , ..)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="toProductionPlanningPage2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>940</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 250, 186);
border-color: rgb(170, 170, 127);
font: 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>suivant </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="capacity">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="BoxCreator" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>921</width>
     <height>121</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="item">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe Script&quot;;
color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>benefice par Tonne:</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Segoe Script&quot;;
color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>Resource consommé de production par Tonnage: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="profit">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="weight">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>40</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 250, 186);
border-color: rgb(170, 170, 127);
font: 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="items">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>590</y>
     <width>251</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe Script&quot;;
color: rgb(98, 74, 181);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="Benefice">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>590</y>
     <width>251</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe Script&quot;;
color: rgb(181, 98, 74);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="resourceOn">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>590</y>
     <width>251</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Segoe Script&quot;;
color: rgb(74, 181, 98);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="text_9">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>540</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Segoe Script&quot;;
color: rgb(98, 74, 181);</string>
   </property>
   <property name="text">
    <string>Items</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="text_10">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>530</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Segoe Script&quot;;
color: rgb(181, 98, 74);</string>
   </property>
   <property name="text">
    <string>Benefice</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="text_11">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>530</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Segoe Script&quot;;
color: rgb(74, 181, 98);</string>
   </property>
   <property name="text">
    <string>Tonne / Resource</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>960</y>
     <width>661</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Segoe Script&quot;;
color: rgb(181, 98, 74);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
